tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/LabelHelper.cs

```diff
@@ -0,0 +1,101 @@
+using Azure.Sdk.Tools.Cli.Services;
+using System.Text;
+using CsvHelper;
+using CsvHelper.Configuration;
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    public interface ILabelHelper
+    {
+        public LabelHelper.ResultType CheckServiceLabel(string csvContent, string serviceName);
+        public string CreateServiceLabel(string csvContent, string serviceLabel);
+        public string NormalizeLabel(string label);
+        public List<LabelData> getLabelsFromCsv(string csvContent);
+    }
+
+    public class LabelHelper(ILogger<LabelHelper> logger) : ILabelHelper
+    {
+        internal const string ServiceLabelColorCode = "e99695"; // color code for service labels in common-labels.csv
+
+        public List<LabelData> getLabelsFromCsv(string csvContent)
```

Comment:
```suggestion
        public static IList<LabelData> GetLabelsFromCsv(string csvContent)
```
This should be a static method because it doesn't rely on any state of a given LabelHelper instance.